### PR TITLE
fix: correct estimated fees so that max spendable works as intended

### PR DESCRIPTION
**Pull request by Sheng-Long**
### 📝 Description

Fix estimated fee so that max spendable works as intended

### ❓ Context

- **Impacted projects**: `` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: libs/ledger-live-common/src/families/hedera/utils.ts [hunk 1]
```diff
@@ -13,7 +13,11 @@ export async function getEstimatedFees(): Promise<BigNumber> {
       url: "https://countervalues.live.ledger.com/latest/direct?pairs=hbar:usd",
     });
 
-    return new BigNumber(10000).dividedBy(data[0]);
+    BigNumber.config({
+      ROUNDING_MODE: BigNumber.ROUND_UP,
+    });
+
+    return new BigNumber(10000).dividedBy(data[0]).dp(0);
```
Comment (haammar-ledger) [inline libs/ledger-live-common/src/families/hedera/utils.ts:20]: Can you please consider using `integerValue` instead.
I am thinking calling `BigNumber.config` could have side effects.

**Review by haammar-ledger — CHANGES_REQUESTED** [review]
Please add a changeset using `pnpm changeset`

**Review by haammar-ledger — APPROVED** [review]

**Comment by Sheng-Long** [comment]
It doesn't seem like the failures here were due to any changes in the PR

**Comment by haammar-ledger** [comment]
@Sheng-Long Apologies we've been having many stability issues with our CI lately, hopefully it should be more stable now.
Can you please rebase on `develop`? Let's give this another go.

**Pull request merged**